ocs_ci/ocs/resources/objectbucket.py

```diff
@@ -222,9 +223,14 @@ def verify_health(self, timeout=60, interval=5):
             logger.error(
                 f"{self.name} did not reach a healthy state within {timeout} seconds."
             )
-            assert (
-                False
-            ), f"{self.name} did not reach a healthy state within {timeout} seconds."
+            obc_obj = OCP(kind="obc", namespace=self.namespace, resource_name=self.name)
+            obc_yaml = obc_obj.get()
+            obc_description = obc_obj.describe(resource_name=self.name)
+            assert False, (
```

Comment:
would be good not to raise an exception here but rather return a value and let the test deal with how to handle the returned value. Think of situations in which a negative result is expected.

Reply:
[We have a function parameter for that](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/conftest.py#L1960)

Reply:
maybe I am missing something but in https://github.com/red-hat-storage/ocs-ci/blob/df461085939bbc7497c793bc40347f5ff598a0ab/tests/conftest.py#L1994 we assert in case verify_heath returns False but here we raise an assertion exception instead of returning bool

Reply:
Actually, we do not assert in case it returns `False` - either we assert it returns `True` and the assertion passes, or it already fails the assertion inside `verify_health` -
https://github.com/red-hat-storage/ocs-ci/blob/df461085939bbc7497c793bc40347f5ff598a0ab/ocs_ci/ocs/resources/objectbucket.py#L226-L228

Reply:
We had a call; I removed the return values entirely and added a custom exception instead of using `assert False`